sycl/include/CL/sycl/xilinx/fpga/opt_decorate_func.hpp

```diff
@@ -53,13 +53,6 @@ void pipeline(T functor) noexcept {
   functor();
 }
 
-// auto pipeline = [] (auto functor) noexcept {
```

Comment:
Is there an issue to put back the modern syntax used by triSYCL instead, now you have increased the C++ version?

Reply:
This has nothing to do with the C++ version unfortunately, this is a limitation enforced by the SYCL specification I believe.

Pipeline isn't just a "function" in this case, it's a global variable, which the SYCL specification restricts and is currently enforced by the SYCL compiler with an error!

Reply:
Interesting.
I think you found a bug in the SYCL specification.
A global `const` variable today is accepted for POD type. I think it should be extended for lambda.
The difference I think is that the lambda can be a local function, and so a local variable. I have the feeling we should allow it for capture-less lambda. Or even for POD-capturing lambda?

Reply:
I created the SYCL specification issue https://gitlab.khronos.org/sycl/Specification/issues/221 to discuss this.